core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -469,6 +468,37 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
         return `+conhost:${hostId} +deleted:false  +(urlmap:* OR basetype:5) ${langQuery} ${archivedQuery} ${keywordQuery} ${identifierQuery}`;
     }
 
+    private getWorflowActionsFn = (item: DotCMSContentlet): Observable<DotCMSPageWorkflowState> => {
```

Comment:
This store is doing so much non-store stuff... you should review that because it will be super hard to maintain or scale.

Reply:
Anything to do here?